editor/src/core/shared/template-path.ts

```diff
@@ -647,6 +669,38 @@ export function isAncestorOf(
   }
 }
 
+function fullElementPathForPath(path: TemplatePath): ElementPath[] {
+  if (isScenePath(path)) {
+    return path.sceneElementPaths
+  } else {
+    return [...path.scene.sceneElementPaths, path.element]
+  }
+}
+
+export function isDescendentOf(
```

Comment:
nitpick, but I think `descendant` is the more commonly used one

Reply:
Oh ffs, I even said this on a call the other day to you! Yep will fix :facepalm:

Reply:
Fixed!